.vsts-ci/phase.yml

```diff
@@ -19,7 +19,7 @@ phases:
 
   queue: ${{ parameters.queue }}
   steps:
-  - powershell:  Install-module pester -Scope CurrentUser -Force -SkipPublisherCheck
```

Comment:
Why are you removing this?

Reply:
Fix whitespace and no publisher check for security (just in case)